tst/hooks/sdk/useActiveSpeakersState.test.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {LogLevel} from "amazon-chime-sdk-js";
+import React from 'react';
+import {renderHook} from '@testing-library/react-hooks'
+import useActiveSpeakersState from '../../../src/hooks/sdk/useActiveSpeakersState';
+import MeetingManager from '../../../src/providers/MeetingProvider/MeetingManager';
+import {MeetingContext} from '../../../src/providers/MeetingProvider';
+
+jest.mock('../../../src/providers/MeetingProvider/MeetingManager');
+
+const mockMeetingManager = MeetingManager as jest.Mock;
+const subscribeToActiveSpeakerSpy = jest.fn();
+const unsubscribeToActiveSpeakerSpy = jest.fn();
+mockMeetingManager.mockImplementation(() => {
+  return {
+    subscribeToActiveSpeaker: subscribeToActiveSpeakerSpy,
+    unsubscribeFromActiveSpeaker: unsubscribeToActiveSpeakerSpy
+  };
+});
+
+beforeEach(() => {
+  // Clear all instances and calls to constructor and all methods:
+  mockMeetingManager.mockClear();
+});
+
+it('should use active speaker', () => {
+  expect(mockMeetingManager).toHaveBeenCalledTimes(0);
```

Comment:
This expect should be a separate test: `should not call meeting manager before use active speaker`